aspnetcore/fundamentals/minimal-apis/security.md

```diff
@@ -0,0 +1,198 @@
+
+There are two strategies for determining user access to resources in the authorization layer:
+
+* Role-based strategies determine a user's access based on the role they are assigned, such as `Administrator` or `User`. For more information on role-based authorization, see [role-based authorization documentation](/aspnet/core/security/authorization/roles).
+* Claim-based strategies determine a user's access based on claims that are issued by a central authority. For more information on claim-based authorization, see [claim-based authorization documentation](/aspnet/core/security/authorization/claims).
+
+In ASP.NET Core, both strategies are captured into an authorization requirement. The authorization service leverages authorization handlers to determine whether or not a particular user meets the authorization requirements applied onto a resource. 
+
+## Enabling authentication in minimal applications
+
+To enable authentication, call [`AddAuthentication`](/dotnet/api/microsoft.extensions.dependencyinjection.authenticationservicecollectionextensions.addauthentication) to register the required authentication services on the app's service provider.
+
+```csharp
+var builder = WebApplication.CreateBuilder(args);
+
+builder.Services.AddAuthentication();
+
+var app = builder.Build();
+
+app.Run();
+```
+
+Typically, a specific authentication strategy is used. In the following sample, the app is configured with support for JWT bearer-based authentication.
+
+```csharp
+using Microsoft.AspNetCore.Authentication.Cookie;
+
+var builder = WebApplication.CreateBuilder(args);
+
+builder.Services.AddAuthentication().AddJwtBearer();
+
+var app = builder.Build();
+
+app.Run();
+```
+
+By default, the [`WebApplication`](/dotnet/api/microsoft.aspnetcore.builder.webapplication) automatically registers the authentication and authorization middlewares if certain authentication and authorization services are enabled. In the following sample, it's not necessary to invoke [`UseAuthentication`](/dotnet/api/microsoft.aspnetcore.builder.authappbuilderextensions.useauthentication) or [`UseAuthorization`](/dotnet/api/microsoft.aspnetcore.builder.authorizationappbuilderextensions.useauthorization) to register the middlewares because [`WebApplication`](/dotnet/api/microsoft.aspnetcore.builder.webapplication) does this automatically after `AddAuthentication` or `AddAuthorization` are called.
```

Comment:
API links should always use xref. Is that right @Rick-Anderson?

I've found https://xrefgen.azurewebsites.net/ makes xref links easy to add.

Reply:
> I've found https://xrefgen.azurewebsites.net/ makes xref links easy to add.

https://rs-xrefs.azurewebsites.net/ is actually much better but it ran out of money and won't work again until Nov 2nd. I've deleted a few resource groups so that shouldn't happen again.

> API links should always use xref. Is that right @Rick-Anderson? Or is the plan to clean them up later?
> 

We prefer them but I accept either. @tdykstra  knows the official rules.

Reply:
The Markdown-style links are easier to produce on the editing side so that's why I tend to rely on them.